src/content/1.7/modules/creation/module-file-structure.md

```diff
@@ -136,24 +140,37 @@ Otherwise it must be regenerated by PrestaShop before the upgrade file applicati
 
 A few details:
 
--   `is_configurable` indicates whether the module has a configuration
-    page or not.
--   `need_instance` indicates whether an instance of the module must be
-    created when it is displayed in the module list. This can be useful
-    if the module has to perform checks on the PrestaShop configuration,
-    and display warning message accordingly.
--   `limited_countries` is used to indicate the countries to which the
-    module is limited. For instance, if the module must be limited to
-    France and Spain, use
-    `<limited_countries>fr,es</limited_countries>`.
+- `is_configurable` indicates whether the module has a configuration page or not.
+- `need_instance` indicates whether an instance of the module must be created when it is displayed in the module list. This can be useful if the module has to perform checks on the PrestaShop configuration, and display warning message accordingly.
+- `limited_countries` is used to indicate the countries to which the module is limited. For instance, if the module must be limited to France and Spain, use `<limited_countries>fr,es</limited_countries>`.
+
+### `logo.png` file
+
+This icon file will be displayed in module listings if present. It needs to be a 32x32 pixel PNG file.
```

Comment:
We use a 128x128 image in example modules 😅 
https://github.com/PrestaShop/example-modules/blob/master/demoextendgrid/logo.png

Reply:
Huh. Native modules are 140x140 which is a very weird size indeed